src/index.js

```diff
@@ -16,8 +16,10 @@ export function h(name, attributes) {
     }
   }
 
-  return typeof name === "function"
-    ? name(attributes || {}, children)
+  return name === null
```

Comment:
@jonlov Why would `name` be null here? I don't want to support that.

Reply:
Now, I see that this is just how you implemented the feature overall. Hmm, thanks for trying. I need to collect my thoughts and see how to go about allowing nulls from subviews. Maybe just return `""`?